pkg/crc/machine/machine.go

```diff
@@ -594,11 +594,11 @@ func Status(statusConfig ClusterStatusConfig) (ClusterStatusResult, error) {
 		ocConfig := oc.UseOCWithConfig(statusConfig.Name)
 		operatorsStatus, err := oc.GetClusterOperatorStatus(ocConfig)
 		if err != nil {
-			result.Success = false
-			result.Error = err.Error()
-			return *result, errors.New(err.Error())
+			logging.Warn(err.Error())
 		}
-		if operatorsStatus.Available {
+		if operatorsStatus == nil {
```

Comment:
Why not check `err != nil` instead?

Reply:
@cfergeau because I am idiot and didn't think about it :(